simpler_eda/numerical_eda.py

```diff
@@ -3,52 +3,51 @@
 import pandas as pd
 
 
-
 def numerical_plot(
     data,
-    xval = None,
-    yval = None,
-    plot_type = "scatter",
-    color = None,
-    title = None ,
-    font_size = 10,
-    colour_scheme = "yellowgreenblue",
-    plot_width = 400,
-    plot_height = 300.
+    xval=None,
+    yval=None,
+    plot_type="scatter",
+    color=None,
+    title=None,
+    font_size=10,
+    color_scheme="yellowgreenblue",
+    plot_width=400,
+    plot_height=300,
     x_transform=False,
     y_transform=False,
 ):
     """
     This function takes in a data frame object, two numeric columns,
     and produces either a scatter or line plot to visualize the
-    relationship of the two features. User can optionally change
-    default arguements for plot-type, color/fill, title, size of text,
+    relationship between the two features. User can optionally change
```

Comment:
please remember to update according to the description on readme if you agreed with my changes.

Reply:
Yes agreed with changes, I have implemented it.